### PR TITLE
Add Dashline

### PR DESCRIPTION
**Pull request by heldev**
### Types of Changes

**What types of changes does your PR introduce?** Put an x in all boxes that apply

- [x] New addition to list
- [ ] Fixing bug in existing item in list
- [ ] Removing item from list
- [ ] Changing structure (organization) of list

### Proposed Changes

Cloud-based password manager with award-winning design, one of the top players.

### PR Checklist

Put an x in the boxes once you've completed each step. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.

- [x] I have read the CONTRIBUTING guide
- [x] I have explained why this PR is important
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by heldev** [comment]
@alichtman I do agree that it's important for a password manager to be open source. But in reality, there is only one OSS Password manager KeePass (and it's variations). I used it for years on many devices (Win, Linux, Mac, Android, iOS) and there are some areas that require improvements. The next step is managers that are closed source but claim that they don't have access to your data (if you lose your pass you lose everything forever). Again I know only one popular manager like this (1Password). And the next step is all other popular managers. So I don't see a difference between LastPass and Dashlane from the security point of view. Dashline has nicer UI, more features, and different pricing model (and actually it has a better history, no big security issue like LastPass so far, but not sure that we should count it because again, they are both build on a model where your data is accessible to the company). 

Do you think we can consider this manager if I fix the typo in the name?

**Comment by alichtman** [comment]
> no big security issue like LastPass

Every "secure" system has issues at some point. What's important is how they're disclosed and handled afterwards. 

I've had some more time to look into it and I'd be alright merging this with the typo fixed. @heldev

**Comment by xcv58** [comment]
Here is why KeePassX is removed: https://github.com/jaywcjlove/awesome-mac/pull/406#issuecomment-399996001

**Comment by alichtman** [comment]
@xcv58 These changes shouldn't be approved. There's a typo that needs to be fixed.

File: README.md [hunk 1]
```diff
@@ -727,7 +727,7 @@ You may come across some non-English characters. Those apps don't have an Englis
 #### Password Management
 
 * [1Password](https://1password.com/) - Cross-platform password management tool.
-* [KeePassX](https://www.keepassx.org) - Light-weight, open-source password management app. [![Open-Source Software][OSS Icon] ![Freeware][Freeware Icon]](https://github.com/keepassx/keepassx)
+* [Dashline](https://www.dashlane.com) - Cloud-based password manager with award-winning design. [![App Store][app-store Icon]](https://itunes.apple.com/us/app/dashlane-password-manager/id552383089)
```
Comment (alichtman) [inline README.md]: `Dashlane` not `Dashline`

**Comment by xcv58** [comment]
Sorry, I miss that. Let me fix it.

**Comment by xcv58** [comment]
Done, it's on master. https://github.com/jaywcjlove/awesome-mac/commit/33e7b7625342976af06f71343eb516532987082b

**Pull request merged**